### ISSUE TITLE
Error getting buddyList. StatusCodeError: 400

### ISSUE THREAD

**Issue by theminer3746** [sqren/fb-sleep-stats#106]
When I run `npm run scrape` I got 
![ebrtastaebng](https://cloud.githubusercontent.com/assets/9322228/24584812/6ea9e33c-17a4-11e7-90ef-4531a66ff360.png)

 "4/2/2017, 12:58:55 PM  - Active users:  11 / 98" but I have at least a few hundreds freinds and at least 40 online

I've read #23 and just like what @bdoner said I make sure my friend are online before starting the scrapper.

Where does the `StatusCodeError: 400` come from? I'm quite sure I get only a subset of my friend because of that.
Any idea?

**Comment by piiwa** [comment]
I get the same issue here

**Comment by canselcik** [comment]
The issue is in the `fb-sleep` library. Facebook returns a 400 response when `clientid` is not provided.

Add these two lines added by https://github.com/sqren/fb-sleep/pull/3/commits/03b89ca8947d11240947bf13b4b534ccba2a361a  to `node_modules/fb-sleep/index.js` and it will be fixed.

Ideally you will want the `client_id` of `messenger.com`. Just inspect the network traffic. Seems to work with `0` but stops working after a while when it is `0`.

**Comment by sorenlouv** [comment]
@canselcik Thanks for making me aware of this. I don't actively maintain this project any longer, but pull requests and fixes are still very welcome. 
Just bumped `fb-sleep` and updated the dependency version in `fb-sleep-stats` to the latest version (2.3.0). 

tldr: pull this repo, run `npm install` and it should work again.
